orion-server/src/main/resources/webapp/src/basic-components/Kafka/BrokersetEntry.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+import {Tab, Tabs, Grid, Box, Link, Typography, Chip} from "@material-ui/core";
+import { Link as RouterLink, Redirect, Route, Switch } from "react-router-dom";
+import PropsTable from "../Commons/PropsTable";
+
+const routes = [
+    {
+        subpath: "assignments",
+        component: PropsTable,
+        label: "Assignments",
+        getData: getAssignmentData,
+        getColumns: getAssignmentColumns,
+    },
+    {
+        subpath: "brokers",
+        component: PropsTable,
+        label: "Brokers",
+        getData: getBrokerData,
+        getColumns: getBrokerColumns,
+    },
+    {
+        subpath: "status",
+        component: PropsTable,
+        label: "Status",
+        getData: getStatusData,
+        getColumns: getStatusColumns,
+    }
+];
+
+function getStatusData(clusterId, rawData) {
+    let brokersetStatus = [];
+    let brokersetData = rawData.brokersetData;
+    brokersetStatus.push({ key: "Broker Count", value: brokersetData.size});
+    return brokersetStatus;
+}
+
+function getStatusColumns() {
```

Comment:
Should this be `getStatsColumns` columns? Are there other places a similar change if needed?